src/backend/aspen/database/models/sequences.py

```diff
@@ -100,6 +100,31 @@ class UploadedPathogenGenome(PathogenGenome):
     sequencing_depth = Column(Float)
     upload_date = Column(DateTime, nullable=False, server_default=func.now())
 
+    def get_stripped_sequence(self) -> str:
```

Comment:
Are there standard fasta-parsing libraries we should be using instead of rolling our own?